home work 1

Julia-92 opened the pull request:
Home Work 1

Julia-92 committed:
home work 1

Julia-92 committed:
Add "registerPage, created_accountPage" ti the Scenario

adudyak commented on .gitignore:
```diff
@@ -1,104 +1,3 @@
-# TernJS port file
-.tern-port
+node_modules
+packege-lock.json
```
typo issue in name

adudyak commented on pages/created_account.js:
```diff
@@ -0,0 +1,8 @@
+
+module.exports = {
+
+  verifyCreatedAccountPageName() {
```
Move this method to register page and remove file

Julia-92 replied:
++

adudyak commented on pages/register.js:
```diff
@@ -0,0 +1,40 @@
+    I.fillField(this.emailField, user.email); 
+    I.fillField(this.phoneField, user.phone); 
+    I.fillField(this.passwordField, user.password); 
+    I.fillField(this.passwordConfirmField, user.passwordConfirm); 
```
Use user.password as input in confirmation field

adudyak commented on pages/register.js:
```diff
@@ -0,0 +1,40 @@
+    I.click(this.subscribeLabel);
+  },
+
+  clickontinueButton() {
```
Check spelling, looks like a duplicate method

Julia-92 replied:
Changed to clickPrivacyPolicyRadio

Julia-92 committed:
Added the clickontinueButton, clickContinueButton to the Scenario

adudyak commented on tests/register_test.js:
```diff
@@ -0,0 +1,25 @@
+const created_account = require("../pages/created_account");
+const { verifyRegisterPageName } = require("../pages/register");
+const register = require("../pages/register");
+
```
Remove rows 1-3

adudyak commented on tests/register_test.js:
```diff
@@ -0,0 +1,25 @@
+    email: Date.now() + '@test.com',
+    phone: "+380932849875",
+    password: "eM8x3xTJ4vzVN",
+    passwordConfirm: "eM8x3xTJ4vzVN",
```
Remove this option for confirm password

adudyak commented on tests/register_test.js:
```diff
@@ -0,0 +1,25 @@
+    registerPage.verifyRegisterPageName();
+    registerPage.fillNewUserForm(NEW_USER);
+    registerPage.clickSubscribeLabel();
+    pause();
```
Remove pause

Julia-92 committed:
Added clickPrivacyPolicyRadio,
removed rows 1-3,
removed option for confirm password,
used the user.password to input in confirmation field,
Moved checking Page Name method to register page and removed the CreatedAccountPage file

Julia-92 committed:
last

Julia-92 committed:
The comma is added to the codecept.conf.js file

Julia-92 committed:
1) Changed places:  registerPage.clickPrivacyPolicyRadio();
    registerPage.clickContinueButton();
2) Remove Pause

Julia-92 committed:
The Xpathes are updated
